core_lib/src/structure/filemanager.cpp

```diff
@@ -374,23 +311,23 @@ Status FileManager::save(const Object* object, QString sFileName)
 
         QString sBackupFile = backupPreviousFile(sFileName);
 
-        Status s = MiniZ::compressFolder(sFileName, sTempWorkingFolder, zippedFiles);
-        if (!s.ok())
+        Status stMiniz = MiniZ::compressFolder(sFileName, sTempWorkingFolder, filesToZip);
+        if (!stMiniz.ok())
         {
-            dd.collect(s.details());
+            dd.collect(stMiniz.details());
             return Status(Status::ERROR_MINIZ_FAIL, dd,
                           tr("Miniz Error"),
                           tr("An internal error occurred. Your file may not be saved successfully."));
         }
         dd << "Zip file saved successfully";
 
-        if (s.ok() && saveLayersOK)
+        if (stMiniz.ok() && saveOk)
```

Comment:
I know we've had this all along, but is the checking the status again needed, we've already determined that it's ok if it comes down here because of the previous check.

Reply:
You are right. I will change it to an `assert()`